python.md

```diff
@@ -119,3 +120,65 @@ use a single thread.
 To enable parallelism with BLIS, one needs to both configure with
 `--enable-threading=openmp` and set the environment variable `BLIS_NUM_THREADS`
 to the number of threads to use, the default is to use a single thread.
+
+## Other common Python packages
+
+### Cmake
+
+On **Ubuntu**:
+
+```
+sudo apt install cmake libssl-dev
+sudo pip3 install scikit-build ninja
+sudo pip3 install cmake
```

Comment:
no you don't want to install cmake via pip this way. I can break the ubuntu package. Why do you need both

Reply:
and why system wide

Reply:
fixed